views/components/adaptor/index.scss

```diff
@@ -0,0 +1,9 @@
+.adaptor {
```

Comment:
it could be possible to define this rule with a 

```
.adaptors > * {
...
}
```

rule in CSS, so that we don't actually need an extra class for `.adaptor` (they would just be the implicit children in the HTML DOM). But I'm ok with either approach.